arrow-libs/optics/arrow-optics-test/build.gradle

```diff
@@ -1,14 +1,27 @@
 plugins {
-    id "org.jetbrains.kotlin.jvm"
+    id "org.jetbrains.kotlin.multiplatform"
     id "org.jlleitschuh.gradle.ktlint"
 }
 
-apply from: "$SUB_PROJECT"
+apply from: "$SUB_PROJECT_MPP"
 apply from: "$PUBLICATION"
-apply from: "$ANIMALSNIFFER"
+apply from: "$ANIMALSNIFFER_MPP"
 
-dependencies {
-    compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$KOTLIN_VERSION"
-    compile project(":arrow-optics")
-    compile project(":arrow-core-test")
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$KOTLIN_VERSION"
```

Comment:
It seems not necessary to add it, but if that's the last info we have let's add it just in case 👍 We'll probably have to update this a couple times given that MPP setup support is only alpha atm.

Reply:
Right, it's not necessary because Gradle solves the dependencies properly now. However it's said that it may change and it's better to use `-common` for the standard library in `commonMain`.

`kotlin.stdlib.default.dependency=false` was removed because of https://youtrack.jetbrains.com/issue/KT-46847 . Kotlin Gradle Plugin adds those dependencies. However, they are added by that plugin as transitive dependencies for all the libraries with specific Kotlin versions. An explicit dependency with `compileOnly` prevents having those transitive dependencies of `kotlin-stdlib` for Arrow libraries.